Makefile

```diff
@@ -53,7 +53,7 @@ e2etests: ## Run the e2e suite against your local cluster
 
 # Run make install-kwok to install the kwok controller in your cluster first
 # Webhooks are currently not supported in the kwok provider.
-apply: verify build ## Deploy the kwok controller from the current state of your git repository into your ~/.kube/config cluster
```

Comment:
Separate verify and apply operations, if not, it costs much time to verify when I need to deploy again after the code changed.

Reply:
Why do you want to skip verify? This does some yq logic to add into CRDs before we apply them on the `make apply` target.

Reply:
Most of the time, I only modify some function logic. Verifying it every time is unnecessary, which makes my development process slow.

This is my original intention for this modification.

Reply:
If someone wants to develop:
they should run:
```sh
make verify
make apply #Run this command again to redeploy if the code has changed
```